content/content-strategy/manage-content-requests/plan-content/Typical-roles-head.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Please rename the file to use a lowercase `t`. The names of the files are case-sensitive. In the index.yml file, there is a reference to `typical-roles-head.md`